vcr/issuer/issuer.go

```diff
@@ -96,6 +97,11 @@ func (i issuer) Issue(ctx context.Context, template vc.VerifiableCredential, opt
 		return nil, errors.New("publishing VC JWTs is not supported")
 	}
 
+	// network published credentials are revoked with revocation credentials
+	if options.Publish && options.WithStatusListRevocation {
```

Comment:
you could publish as did:web to a did:nuts subject over the network? So we don't really need the check?

Or we just disable publishing did:web VCs (issuer or subject alike), and only support (or always support) StatusList2021 when issuing did:web (for now), then we don't need additional checks.

Reply:
I'd ignore `visibility` and `PublishToNetwork` for `did:web`-related VCs